tests/production/DeleteUserSettingsFromCouchTests.js

```diff
@@ -96,7 +96,7 @@ fluid.defaults("gpii.tests.productionConfigTesting.deleteUserSettings", {
         {
             // remove the loaded GPII key "os_gnome" and its prefs safe
             func: "{getOsGnomeKey}.send",
-            args: [null, { port: "5984" }]
+            args: [null, { port: gpii.tests.productionConfigTesting.couchdbUrl.port }]
```

Comment:
Same as above. The same comment applies to changes below at line 114, 129, 136, 143, 150, 158, 165, 172, 179, 187, 202.